tests/src/test/scala/scalaz/MapTest.scala

```diff
@@ -972,6 +972,7 @@ object MapTest extends SpecLite {
   checkAll(order.laws[Int ==>> Int])
   checkAll(monoid.laws[Int ==>> Int])
   checkAll(band.laws[Int ==>> ISet[Int]])
+  checkAll(semilattice.laws[Int ==>> ISet[Int]])
```

Comment:
We can remove `band.laws[Int ==>> ISet[Int]]` if `trait Semilattice extends Band` and `semilattice laws` include `band laws`